presto-main/src/main/java/io/prestosql/sql/relational/RowExpressionVisitor.java

```diff
@@ -17,6 +17,8 @@
 {
     R visitCall(CallExpression call, C context);
 
+    R visitSpecialForm(SpecialForm specialForm, C context);
```

Comment:
Seems like it would be more consistent to call this new class `SpecialFormExpression`.